CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 4.2.0
+#### API updates:
+
+- Added new method `setMixpanelDistinctId` as a convenience method for setting the required attribute for the Mixpanel integration #1397
+
+- Async promotional offer methods no longer include the `get` prefix #1405
```

Comment:
🤔 let's be more specific since they're API changes. 
Could we include something like: 
- `foo` has been deprecated in favor of `bar`